.github/workflows/armv7l_wheels.yml

```diff
@@ -0,0 +1,60 @@
+name: armv7l wheels
+
+on:
+  push:
+  pull_request:
+  create:
+  schedule:
+    - cron:  '23 1 * * *'
+
+env:
+  SERVER_IP: '159.203.106.198'
+
+jobs:
+  armv7l:
+    runs-on: ubuntu-18.04
+    if: github.event_name == 'schedule' || (github.event_name == 'create' && github.event.ref_type == 'tag') || contains(github.event.head_commit.message, '[build wheel]') || contains(github.event.head_commit.message, '[build wheel armv7l]') || contains(github.event.pull_request.title, '[build wheel]') || contains(github.event.pull_request.title, '[build wheel armv7l]')
+    strategy:
+      matrix:
+        docker_images: ['balenalib/armv7hf-debian:stretch', 'balenalib/armv7hf-debian:buster']
+    steps:
+    - uses: actions/checkout@v1
+    - name: Make ${{ matrix.docker_images }} wheel
+      run: |
+        source .ci/ubuntu_ci.sh
+        generate_armv7l_wheels ${{ matrix.docker_images }}
+    - name: Rename wheels
+      if: github.event.ref_type != 'tag'
+      run: |
+        source .ci/ubuntu_ci.sh
+        rename_wheels
+    - name: Upload wheels to server
+      if: github.event_name != 'pull_request'
+      env:
+        UBUNTU_UPLOAD_KEY: ${{ secrets.UBUNTU_UPLOAD_KEY }}
+      run: |
+        source .ci/ubuntu_ci.sh
+        upload_file_to_server "$SERVER_IP" "armv7l/kivy/"
```

Comment:
Now that we also do armv61 wheels, I changed the name to be more reflective of the platform, so it should be `"raspberrypi/kivy/"`. See https://kivy.org/downloads/ci/raspberrypi/kivy/.

Reply:
But this should work on any armv6l platform, as far as I understand? The reason why the Raspberry Pi needed a special wheel before was because it used the proprietary Broadcom drivers.

Reply:
I don't think that's correct. If you read their FAQ they explicitly say that it may not work on other armv6 platforms.

The reason is that unlike manylinux, where before publishing a wheel we use auditwheel to gather all the OS level deps and include them in the wheel - except for a PEP specified list of .so files that are considered part of the OS and must be available by default on any OS that supports manylinux. Here, we simply assume any OS level deps are available because the user explicitly installs them with apt. So this wheel makes the specific assumption that it has been done, and it's specific to the OS/OS family on which this has been built.

So we cannot assume it will work on other armv6 platforms, e.g. android, except by coincidence, until there's support for these platforms on auditwheel. That's why they are/won't be available on pypi: https://github.com/pypa/warehouse/issues/3668.

Reply:
Ahh I see. I'll change the files.

Reply:
See: b31d6baeaa18b1ed39a92f455f872a38299ad00c